.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+#### Issue Number
+ISSUE #
+<!-- Please Mention the issue number as  ISSUE #(Issue Number)
+Example:
+ISSUE #5
+-->
+
+#### Describe the changes you've made
+A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!
```

Comment:
I think, It's better to keep line `9` in Comments since It need not be displayed out